src/EnergyPlus/Construction.cc

```diff
@@ -1919,6 +1920,20 @@ void ConstructionProps::reportTransferFunction(EnergyPlusData &state, int const
     }
 }
 
+void ConstructionProps::reportLayers(EnergyPlusData &state)
+{
+    // Report the layers for each opaque construction in predefined tabular report
+    // J. Glazer March 2024
+    auto &opd = state.dataOutRptPredefined;
+    if (state.dataOutRptPredefined->pdchOpqConsLayCol.size() > 0) {
+        for (int i = 1; i <= this->TotLayers; ++i) {
+            int layerIndex = this->LayerPoint(i);
+            auto const *thisMaterial = state.dataMaterial->Material(layerIndex);
```

Comment:
Hmm... are materials held as pointers?  If so, this is great.  If not, this is still technically right, but you could use a reference instead like we do all over.  I'm assuming it must be held as a pointer or you wouldn't have done that.

Reply:
Thanks, I just copied that line without thinking. I fixed it to a normal reference.